CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+
+
+2019-1-29
+=======================
+
+Features & Improvements
+-----------------------
+
+- Allow `serialize_data.py` accept path for proposed endpoints docs as command line argument. [#1](https://github.com/CuriousLearner/API-stub-generator/pull/4)
```

Comment:
Instead of url to the issue, you can add your name and link to your github profile.